validate.py

```diff
@@ -207,6 +208,15 @@ def determine_if_mergeable():
   for user, state in sorted(reviews.items()):
     print ('  %s: %s' % (user, state))
 
+  # As the PR gets older, require more and more rejections to "veto" it,
+  # overriding any non-unanimous majority approvals.
+  # Thus, a newly opened PR can be blocked with a single veto (0 days + 1)
+  # A PR that has been open for 3 days requires four rejections to be vetoed.
+  # (Note that this is the number of days since the PR was opened, not since its last change.)
+  pr_creation_timestamp = pr_created_at_ts(pr_json)
+  pr_age_in_days = math.floor(seconds_to_days(seconds_since(pr_creation_timestamp)))
```

Comment:
There are utility functions this can leverage

Reply:
hm? Do you mean you'd like to see me write some, or that some exist? I didn't see anything like `days_since`.

Reply:
The `math.floor` isn't needed, since `seconds_to_days` already truncates.

Reply:
I might be wrong about what exactly is in there, or getting confused with an unmerged PR. If you don't see anything, go ahead and resolve this. I had the impression you hadn't re-evaluated since your initial commit, but I may be mistaken.

Reply:
I probably need to give it another sober pass, I had like 10 minutes of downtime at work in which I did this. I need to test it locally.